src/software/thunderscope/thread_safe_buffer.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, buffer_size, protobuf_type, log_overrun=False):
         self.protos_dropped = 0
         self.last_logged_protos_dropped = 0
 
-    def get(self, block=False, timeout=None):
+    def get(self, block=False, timeout=None, return_default=True):
```

Comment:
New parameter should be documented. 
```suggestion
    def get(self, block=False, timeout=None, return_cached=True):
```

Reply:
done